internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestVeniceHelixAdminWithIsolatedEnvironment.java

```diff
@@ -311,4 +315,43 @@ public static boolean resourceMissingTopState(SafeHelixManager helixManager, Str
     }
     return false;
   }
+
+  @Test
+  public void testDeadStoreStatsInitialization() {
+    int newAdminPort = controllerConfig.getAdminPort() + 100;
+    PropertyBuilder builder = new PropertyBuilder().put(controllerProps.toProperties())
+        .put("admin.port", newAdminPort)
+        .put("controller.dead.store.endpoint.enabled", true)
+        .put("controller.dead.store.stats.class.name", MockDeadStoreStats.class.getName());
```

Comment:
Does this test even do anything? We are not enabling `controller.dead.store.endpoint.enabled` or `controller.dead.store.stats.pre.fetch.enabled`... If we want to actually verify that the `MockDeadStoreStats`'s `preFetchStats` is called we should add some implementation and methods so we can assert that. e.g. a boolean named `hasCache` that initializes to false. Set to true in preFetchStats so you can verify. If you are setting VeniceHelixAdmin's deadStoreStats directly via some mock you can also use `verify` to ensure certain method is invoked for a given mock.